File: php/h00.associative.php
#!/usr/bin/php
<?php
$ages[bob]=34;
$ages[ed]=58;
$ages[steve]=32;
$ages[ralph]=23;
$ages[deb]=46;
$ages[kate]=19;
 
echo "Keys (names): ", join(" ", (array_keys($ages))), "\n";
echo "Values (ages): ", join(" ", (array_values($ages))), "\n";
?>

File: perl/h00.associative.pl
#!/usr/bin/perl -w
use strict;
my %ages;           # declare hash
  
# add key/value one by one
$ages{bob}=34;     
$ages{ed}=58;
$ages{steve}=32;
$ages{ralph}=23;
$ages{deb}=46;
$ages{kate}=19;
  
# print keys and values
print "Keys (names): ", join (" ", keys %ages), "\n";
print "Values (ages): ", join (" " , values %ages), "\n";
